src/containers/ProfileDetails/components/SSID/index.js

```diff
@@ -36,6 +37,8 @@ const SSIDForm = ({
     }
   };
 
+  const sortedRadios = sortRadioTypes(Object.keys(radioTypes || []));
```

Comment:
change the order of radioTypes in wlan-cloud-ui and CMAP then you won't have to sort. https://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order